<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QDialog" name="SimulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Симуляция параметров</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="tempGroup">
     <property name="title">
      <string>Температура</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="tempSpin">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-273.149999999999977</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>25.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="tempUnitCombo">
        <item>
         <property name="text">
          <string>°C (Цельсий)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>°F (Фаренгейт)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>K (Кельвин)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="humidityGroup">
     <property name="title">
      <string>Влажность</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="humiditySpin">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pressureGroup">
     <property name="title">
      <string>Давление</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QDoubleSpinBox" name="pressureSpin">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200000.000000000000000</double>
        </property>
        <property name="value">
         <double>101325.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pressureUnitCombo">
        <item>
         <property name="text">
          <string>Па (Паскаль)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>мм рт.ст.</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
